lib/aptible/auth/organization.rb

```diff
@@ -4,6 +4,7 @@ class Organization < Resource
       has_many :roles
       has_many :users
       has_many :invitations
+      has_many :whitelist_memberships
```

Comment:
Q: Should we also add `has_many: saml_configurations`?

Reply:
We don't model it that way in auth. Each organization only has one saml_configuration and we don't store the foreign key in the organization. We just directly look-up the configuration based on either the organization ID or the SSO login handle (which is stored in the saml_configuration). It saves the db query for the organization row since we expect the "human friendly" handle to the most common way.

I don't feel strongly about NOT doing this, it just wasn't needed for the testing we doing and we don't anticipate adding CLI configuration for SSO.